src/Components/styles.css

```diff
@@ -49,25 +50,95 @@ img {
   color: rgb(0, 132, 137);
 }
  #leftScroll {
+  align-self: center;
   border-style: none;
   width: 20px;
   position: fixed;
-  height: 350px;
+  height: 175px;
   text-align: center;
   font-size: 30px;
-  font-weight: 2000;
+  font-weight: 1000;
   color: slategrey;
   background: transparent;
+  -webkit-transform:scale(1,2);
+  z-index: 1;
 }
 
 #rightScroll {
+  align-self: center;
   border-style: none;
   color: slategrey;
   width: 20px;
   position: fixed;
   font-size: 30px;
-  height: 350px;
+  height: 175px;
   font-weight: 1000;
   right: 0px;
   background: transparent;
+  -webkit-transform:scale(1,2);
+  z-index: 1;

```

Comment:
Left scroll or right scroll seem to have a lot of similar css styles. Consider giving them a similar class name, and adding the common styles to that class.